notes/returned-mail.vbs

```diff
@@ -319,8 +342,13 @@ ELSE
 		EMReadScreen open_cash2, 2, 8, 43
 		IF open_cash1 <> "  " THEN EMWriteScreen "3", 6, 58 'Enter code "3" (Refused/Failed Required Info)'
 		IF open_cash2 <> "  " THEN EMWriteScreen "3", 8, 58 'Enter code "3" (Refused/Failed Required Info)'
-       	If snap_case = TRUE THEN EMWriteScreen "4", 12, 58 'Enter code "4" (Refused/Failed (FS Only))'
-       	IF grh_case = TRUE THEN EMWriteScreen "3", 10, 58 'Enter code "3" (Refused/Failed Required Info)'
+       	IF snap_case = TRUE THEN
```

Comment:
PACT functionality failed on training case #305410

Reply:
the if statement needed to be updated